bundle/lib/systemd/system/rke2-agent.env

```diff
@@ -0,0 +1 @@
+HOME=/root
```

Comment:
This is likely to cause issues with the RKE2-packaging repo for RPMs, because it will "override" the specific RKE2_SELINUX=true change we made: https://github.com/rancher/rke2-packaging/blob/master/rpm/centos7/agent/rke2-agent.env

Reply:
According to the spec file the RPM's .env file goes to `%{_sysconfdir}/sysconfig/rke2-agent`. Both files are loaded independently via EnvironmentFile in the unit, and we're not unsetting RKE2_SELINUX in this one, so I don't see how it would override it.

Reply:
I was concerned not because it's ending up as `sysconfig/rke2-agent` but because the RPM build process performs an `untar` of the contents to the `_builddir` and then cherry-picks content out of the `_builddir`; however, looking more closely at the RPM build process, it looks as if we will utilize the copy of `rke2-agent.env` from the `_sourcedir` rather than `_builddir` and as such, this should be OK.